crucible-mir/src/Mir/PP.hs

```diff
@@ -328,7 +328,7 @@ instance Pretty ConstVal where
     pretty (ConstRawPtr a) = pretty a
     pretty (ConstStruct fs) = pretty "struct" <> list (map pretty fs)
     pretty (ConstEnum v fs) = pretty "enum" <> list ((pretty "variant" <+> pretty v) : map pretty fs)
-    pretty (ConstSlice cs)  = list (map pretty cs)
+    pretty (ConstSliceRef a len) = pretty "&" <> pr_id a <> (viaShow len)
```

Comment:
There was some reason I did that rather than just ignore the value, but I can't remember what and I agree the resulting output is nonsense. I think, since this is for prettyprinting rather than ast-dumping, that it should just skip the length. Updated in 3444637b455edf15c14963403aa69cd2f4192f5e